docs/JestObjectAPI.md

```diff
@@ -321,6 +321,25 @@ _Note It is recommended to use [`jest.mock()`](#jestmockmodulename-factory-optio
 
 Returns the actual module instead of a mock, bypassing all checks on whether the module should receive a mock implementation or not.
 
+Example:
+
+```js
+jest.mock('../myModule', () => {
+  // Require the original module to not be mocked...
+  const originalModule = jest.requireActual(moduleName);
```

Comment:
What is `moduleName`? Is it `../myModule`? Is it `myModule`? Should be clarified.

Reply:
`'../myModule'`. Wanna send a PR?

Reply:
Thanks. Yes, happy to.

Reply:
Done -> https://github.com/facebook/jest/pull/10255